test/jdk/java/util/stream/test/org/openjdk/tests/java/util/stream/WhileOpTest.java

```diff
@@ -47,7 +47,6 @@
 /*
  * @test
  * @bug 8071597 8193856
- * @run main/timeout=240
```

Comment:
The original change which introduced this `main/timeout` appears to be https://bugs.openjdk.org/browse/JDK-8134459. Going through the RFR of that change https://mail.openjdk.org/pipermail/core-libs-dev/2018-January/050878.html it appears that in addition to this timeout being introduced there was an additional change done to help improve the duration of this test. Plus there's also a comment in that JBS issue that the timeout hadn't been observed for a while (except locally).

I think all that explains why this test hasn't been timing out in the presence of this no-op directive.